src/table/Schema.cc

```diff
@@ -705,6 +706,13 @@ int Schema::compare(Schema const &other, int flags) const {
     return result;
 }
 
+std::size_t Schema::hash_value() const noexcept {
+    std::size_t result = 17;
+    auto hasher = [&result](auto const &item) { result = utils::hashCombine(result, item.key); };
```

Comment:
I don't know how we survived before C++14 and universal lambdas :smile:.

Reply:
Actually, I originally wrote this using an explicit functor similar to [`Stream`](https://github.com/lsst/afw/blob/master/src/table/Schema.cc#L727-L738). The only thing the universal lambda really does for you here is let you put the class template inside the function definition.

Reply:
Yup, I just really like being able to do that :slightly_smiling_face: